Lib/test/test_math.py

```diff
@@ -1202,6 +1266,277 @@ def testLog10(self):
         self.assertEqual(math.log(INF), INF)
         self.assertTrue(math.isnan(math.log10(NAN)))
 
+    def testSumProd(self):
```

Comment:
```suggestion
    @unittest.expectedFailure  # TODO: RUSTPYTHON
    def testSumProd(self):
```
Adding an expectedFailure decorator can defer resolving actual implementation of them to next patch.